wordcloud.py

```diff
@@ -1,30 +1,28 @@
+"""Create word cloud tools."""
 import operator
-import random
 import string
 
-FILENAME = "initial_text.txt"
 WORDS_WITH_VALUE = ["word", "cloud", "punctuation"]
 
 
-def main():
-    """Produce words suitable for processing into a word cloud using random model selection."""
-    raw_text = read_file(FILENAME)
+def main(source_text, model_selection="occurrence"):
+    """Produce words suitable for processing into a word cloud."""
+    raw_text = read_file(source_text)
     processed_list = process_string(raw_text)
-    model_selection = random.randint(1, 4)
-    if model_selection == 1:  # Creates a dictionary based on occurrence of words within list
-        word_to_occurrence = determine_occurrences(processed_list)
-        word_to_occurrence = insert_name(word_to_occurrence, "**OCCURRENCE BASED MODEL**")
-    elif model_selection == 2:  # Creates a dictionary based on value applied to word
-        word_to_occurrence = allocate_values(processed_list)
-        word_to_occurrence = insert_name(word_to_occurrence, "**WORD-VALUES BASED MODEL**")
-    elif model_selection == 3:  # Creates a dictionary based on length of word
-        word_to_occurrence = determine_length(processed_list)
-        word_to_occurrence = insert_name(word_to_occurrence, "**WORD-lENGTH BASED MODEL**")
+    if model_selection == "occurrence":  # Creates a dictionary based on occurrence of words within list
```

Comment:
Yes.  I have changed where the comment is.  The model selected is now a constant and the option descriptions are an in-line comment next to it.

Reply:
That's better. 

I suggest describing each model in the docstring so that is clear exactly what they do. For example, how does alphabetical work? It creates a dictionary of words to their ordered position when sorted alphabetically.

Reply:
I'm trying to improve them without getting too long.  I'll have to think about this a bit.

Reply:
Why not use what you've got already?

Something like:
```
"""Produce a dictionary of words suitable for processing into a word cloud using the specified model.

Available models:
"occurrence": Creates a dictionary based on occurrence of words within list.
"length": Creates a dictionary based on occurrence of words within list.
"value": Creates a dictionary of words to their index when sorted alphabetically. <--- My suggestion
"""
```

Reply:
OK thank you.  I've used that format for the docstring.  Looks good.